libinstpatch/libinstpatch.def

```diff
@@ -1,6 +1,9 @@
 LIBRARY
 EXPORTS
 
+ipatch_close
+ipatch_deinit
```

Comment:
Why are you exporting `ipatch_deinit`?

Reply:
May be ipatch_deinit() could be also a future API  for a future version ?

Reply:
I don't see how it could be useful. We have `ipatch_close()`. I don't see a need for making `ipatch_deinit()` public.

Reply:
Yes, `ipatch_close()` is more complete and should be sufficient. I will revert `ipatch_deinit()` to non public.